examples/catalog/lib/app_bar_bottom.dart

```diff
@@ -76,7 +82,7 @@ class _AppBarBottomSampleState extends State<AppBarBottomSample> with SingleTick
 }
 
 class Choice {
-  const Choice({ this.title, this.icon });
+  const Choice({this.title, this.icon});
```

Comment:
In the Flutter sources we consistently include spaces before and after the first and last parameters (for one line parameter lists) because it makes the parameter list beginning and end look a little less like line noise.

Reply:
I'll be happy to fix the 29 cases (5%) that aren't formatted correctly. Looks like they're mostly tests.

Reply:
Ok putting the horse back in the barn. I've tweaked my experimental formatter to match the flutter style for this case. I'm only concerned that we aren't left with cases where running the default dart formatter will result is deeply confusing code like the cases of `if` and `for` expressions and the cases of missing trailing commas.

Reply:
If you view the latest patch in this cl you will see the patch that adds all the fixes for this case.